walkthroughs/zap/zaphud.md

```diff
@@ -0,0 +1,51 @@
+---
+label: Running the Zap HUD tutorial
+icon: code-square
+order: 100
+---
+
+# Application Security Community Walkthrough
+
+## Zap HUD
+Zap's HUD functionality is an overlay that provides access to tools you can use while testing a site dynamically.
+
+Some of the benefits of using the HUD over one of the static scripts are:
+- First contact with a web application.  When you first encounter a web application you will not have existing scripts.  The HUD can help you do some basic analysis to determine what should be scripted specifically for the future.
+- Authentication.  Since ZAP is a proxy and the HUD is an overlay for sites running through that proxy you can use the dynamic assessment capabilities to manually authenticate, especially when the authentication information is too sensitive to leave in a script.
+
+### Zap HUD tutorial
+Zap HUD already has a good built-in tutorial for what features it supplies and some practices exercises on using various features it provides. 
+
+Rather than a feature by feature walkthrough this walkthrough will be how to access that tutorial.
+Note that any flags in that tutorial are not part of the (Hacker's Challenge)[https://www.saintcon.org/con-hackerschallenge/]
+
+#### Manual Explore
+From the ZAP home screen, chose the option "manual explore".  This will take to to a screen such as this:
+![](/static/zap/manual-explore.png)
+
+##### Setup
+
+###### URL to Explore
```

Comment:
I would like to add a link to a URL we know will be running for people to access, but am not sure what will be available at the moment.

Reply:
Working on getting a URL atm